README.md

```diff
@@ -1,249 +1,25 @@
-
-You can use the same script to archive directories in the `shared` directory, by modifying 
-the inputs slightly:
-
-- Set `--basedir=/home/jovyan/shared/`, (or `--basedir=/home/jovyan/shared-readwrite/`
-  if you want to be able use the `--delete` flag).
-- Create a file with a list of directories in the `shared` directory you want to archive,
-  and pass it to the `--usernames-file` argument.
-- Set `--object-prefix="archives/_shared/` to put the archives in the `_shared` subdirectory
-  in the archive bucket.
-
-E.g.:
-
-```
-python3 archive-home-dirs.py \
-    --archive-name="archive-$(date +'%Y-%m-%d')" \
-    --basedir=/home/jovyan/shared/ \
-    --bucket-name=openscapeshub-prod-homedirs-archive \
-    --object-prefix="archives/_shared/" \
-    --usernames-file=/home/jovyan/shared-to-archive.txt \
-    --temp-path=/home/jovyan/archive-staging/
-```
-
-By default, archives (`.tar.gz`) are created in your `/tmp` directory before
-upload to the S3 bucket. The `/tmp` directory is cleared out when you shut down
-the Hub. However, `/tmp` has limited space (80GB shared by up to four users on a
-single node), so if you are archiving many large directories, you will likely
-need to specify a location in your `$HOME` directory by passing a path to the
-`--temp-path` argument. The script will endeavour to clean up after itself and 
-remove the `tar.gz` file after uploading, but double check that directory
-when you are finished or you may have copies of all of the other user
-directories in your own `$HOME`!
-
+This repository is a place for us to develop draft policies for NASA-Openscapes. 
+The current version of our access policies can be found in the [Earthdata Cloud Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/policies-admin/).
+
+## Scripts and tools
+We also use this repository to store code we use to help understand and implement
+our policies:
+
+- Archive user directories: 
+  - [Data Storage Policy](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/policies-admin/data-policies.html)
+  - [Python script](scripts/archive-home-dirs.py) and [documentation](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/policies-admin/data-policies.html)
+- Monitoring cloud usage costs:
+  - [Policy](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/policies-admin/#monitoring-cloud-usage-costs)
+  - Automated report (coming soon)
+
+## GitHub Discussions
+We’re using [Discussions](https://github.com/NASA-Openscapes/2i2cAccessPolicies/discussions) as a place to connect with other members of our community. We hope that you:
+
+- Ask questions you’re wondering about.
+- Share ideas.
+- Engage with other community members.
+- Welcome others and are open-minded. Remember that this is a community we build together 💪.
+
+The thread [What's happening on the NASA Openscapes Hub!?](https://github.com/NASA-Openscapes/2i2cAccessPolicies/discussions/2) is a great place to share the 
```

Comment:
```suggestion
[**What's happening on the NASA Openscapes Hub!?**](https://github.com/NASA-Openscapes/2i2cAccessPolicies/discussions/2) - please share stories big and small about your experiences using the Hub for science and teaching. These stories are important to communicate the value of hosting cloud resources so that we can continue to do so. Thank you!
```